src/ubx-message.toit

```diff
@@ -60,14 +73,250 @@ class Message:
+      0x09: "ODO",        // M8+
+      0x10: "RESETODO",   // M8+
+      0x11: "VELECEF",
+      0x12: "VELNED",
+      0x13: "HPPOSECEF",  // M8+
+      0x14: "HPPOSLLH",   // M8+
+      0x20: "TIMEGPS",
+      0x21: "TIMEUTC",
+      0x23: "TIMEGLO",    // M8+
+      0x24: "TIMEBDS",    // M8+
+      0x25: "TIMEGAL",    // M8+
+      0x26: "TIMELS",     // M8+
+      0x28: "NMI",        // M8+
+      0x30: "SVINFO",
+      0x31: "DGPS",
+      0x32: "SBAS",
+      0x35: "SAT",        // M8+
+      0x39: "GEOFENCE",   // M8+
+      0x3B: "SVIN",       // M8+
+      0x3C: "RELPOSNED",  // M8+
+      0x3D: "EELL",       // M8+
+      0x42: "SLAS",       // M8+
+      0x60: "AOPSTATUS",
+      0x61: "EOE",        // M8+
+    },
+
+    // RXM (0x02)
+    RXM: {
+      0x10: "RAW",     // 6-series
+      0x11: "SFRB",    // 6-series
+      0x13: "SFRBX",   // M8+
+      0x14: "MEASX",   // M8+
+      0x15: "RAWX",    // M8+
+      0x20: "SVSI",
+      0x30: "ALM",
+      0x31: "EPH",
+      0x32: "RTCM",    // M8+
+      0x41: "PMREQ",
+      0x59: "RLM",     // M8+
+      0x61: "IMES",    // M8+
+    },
+
+    // TIM (0x0D)
+    TIM: {
+      0x01: "TP",
+      0x03: "TM2",
+      0x04: "SVIN",
+      0x06: "VRFY",    // 6-series
+      0x11: "DOSC",    // M8+
+      0x12: "TOS",     // M8+
+      0x13: "SMEAS",   // M8+
+      0x15: "VCOCAL",  // M8+
+      0x16: "FCHG",    // M8+
+      0x17: "HOC",     // M8+
+    },
+
+    // SEC (0x27) — M8+
+    SEC: {
+      0x03: "SEC-UNIQID",
+    },
+
+    // AID (0x0B) — legacy assistance (6 & M8)
+    AID: {
+      0x01: "INI",
+      0x02: "HUI",
+      0x30: "ALM",
+      0x31: "EPH",
+      0x33: "AOP",
+      0x50: "ALP",   // 6-series
+      0x10: "DATA",  // 6-series
+      0x32: "ALPSRV" // 6-series
+    },
+
+    // ESF (0x10) — external sensor fusion
+    ESF: {
+      0x02: "MEAS",  // 6-series LEA-6R / M8 ESF-MEAS (different payloads)
+      0x03: "RAW",   // M8+
+      0x10: "STATUS",// 6-series LEA-6R / M8 ESF-STATUS
+      0x14: "ALG",   // M8+
+      0x15: "INS",   // M8+
+    },
+
+    // HNR (0x28) — M8+
+    HNR: {
+      0x00: "PVT",
+      0x01: "ATT",
+      0x02: "INS",
+    },
+
+    // LOG (0x21) — M8+
+    LOG: {
+      0x03: "ERASE",
+      0x07: "CREATE",
+      0x0B: "INFO",
+      0x0D: "RETRIEVE",
+      0x0E: "RETRIEVEPOS",
+      0x0F: "RETRIEVESTRING",
+      0x10: "FINDTIME",
+      0x11: "BATCH",
+    },
+
+    // MGA (0x13) — M8+ multi-GNSS assistance (index only; many subtypes)
+    MGA: {
+      0x00: "GPS",
+      0x02: "GAL",
+      0x03: "BDS",
+      0x05: "QZSS",
+      0x06: "GLO",
+      0x20: "ANO",
+      0x40: "INI",
+      0x60: "ACK",
+      0x80: "DBD",
+    },
+  }
 
   static INVALID-UBX-MESSAGE_ ::= "INVALID UBX MESSAGE"
   static RESERVED_ ::= 0
 
+  /**
+  Represents the minimum protocol version for the message type.
+
+  Devices must support at least this protocol version to use the message.
+
+  Todo: convert to semver for later ease of use...?
```

Comment:
Not something the user should care about -> not a toitdoc comment.

I would keep it as string, unless there are going to be lots of semver comparisons.